C-Plus-Plus/graphs/Check_for_bipartite_graph.cpp

```diff
@@ -1,110 +1,85 @@
-/*
-Bipartite Graph Check
-----------------------
-Problem:
-Divide all vertices of the graph in two sets such that
-all the edges of the graph are from set1 to set2
+/* this the program of the bipartite check
```

Comment:
I don't see the point of changing the algorithm description to something else just out of blue.